.ci/jenkins/build-test-deploy/Jenkinsfile

```diff
@@ -78,6 +78,27 @@ pipeline {
                 archiveArtifacts artifacts: 'test_results/**/*.log', fingerprint: true
             }
         }
+        stage('Scan image(s) for vulnerabilities'){
+            environment{
+                TMPDIR = '/data/tmp'
+            }
+            steps{
+                echo "Scanning ${DOCKER_IMAGE_PREFIX} docker images with tag ${DOCKER_TAG} for patchable vulnerabilities"
+                sh label: 'Update Grype DB',
+                   script: "grype db update"
+                script{
+                    DOCKER_IMAGE_SUFFIXES.tokenize(',').each { DOCKER_IMAGE_SUFFIX ->
+                        echo "Checking ${DOCKER_IMAGE_PREFIX}/${DOCKER_IMAGE_SUFFIX} docker image with tag ${DOCKER_TAG}"
+                        sh label: "Check ${DOCKER_IMAGE_PREFIX}/${DOCKER_IMAGE_SUFFIX} OPERA Docker image",
+                           script: """
+                           grype ${DOCKER_IMAGE_PREFIX}/${DOCKER_IMAGE_SUFFIX}:$DOCKER_TAG --only-fixed -o template -t .ci/jenkins/html.tmpl --file test_results/${DOCKER_IMAGE_SUFFIX}/vulnerabilities.html
```

Comment:
Let's include grype in the name of the output html so its clear what the source is, something like `grype_vulnerabilities.html` or `grype_scan_results.html`

Reply:
Thanks Scott, made the suggested changes